openstates/mo/events.py

```diff
@@ -0,0 +1,155 @@
+import re
+import datetime
+import requests
+
+from lxml import html
+from openstates.utils import LXMLMixin
+from billy.scrape.events import EventScraper, Event
+
+
```

Comment:
We zap all double and trip white spaces in our code as a rule -- always feely fussy but over the long term I like the effect.